desktop-widgets/tableview.cpp

```diff
@@ -60,7 +60,10 @@ TableView::TableView(QWidget *parent) : QGroupBox(parent)
 		iconSize = btnSize - 2*min_gap;
 	}
 	plusBtn->setIconSize(QSize(iconSize, iconSize));
+#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
```

Comment:
And if you remove it unconditionally?

Reply:
can you try it on your Linux system without it. For me it looks better on Linux with this line... and I would love to figure out how to make it look ok **and** work. But of course you are right, we could just remove it, period.

Reply:
Yes - definitely looks super ugly without this line. That's a bit infuriating. Perhaps ask an Qt expert?

Reply:
According to a quick search, you should use a `QToolButton` instead of a `QPushButton`. However, that made it look even worse. Giving up - sorry.

Reply:
yeah - I got that far as well.
I figured for 4.9.7 I'd go with the compromise in this PR, and then I'll think about a better solution afterwards.
Or maybe I find time this week to poke at it some more. But I think this one is "good enough". Any other comments, or good to merge?

Reply:
The build system stuff is Chinese to me. So fine with me.